### PR TITLE
fix in example (vi): utxoEntrySize is 200

### PR DESCRIPTION
**Pull request by francolq**

### CODE REVIEW

File: doc/explanations/min-utxo-alonzo.rst [hunk 1]
```diff
@@ -60,7 +60,7 @@ Recall that
 +--------------------------+-----------------+-----------------+-------------------+------------------+------------------+---------------------------------+
 | size of value            | 11              | 12              | 15                | 16               | 17               | 173                             |
 +--------------------------+-----------------+-----------------+-------------------+------------------+------------------+---------------------------------+
-| ``utxoEntrySize``        | 38              | 39              | 42                | 43               | 44               | 20                              |
+| ``utxoEntrySize``        | 38              | 39              | 42                | 43               | 44               | 200                              |
```
Comment (JaredCorduan) [inline doc/explanations/min-utxo-alonzo.rst]:
```suggestion
| ``utxoEntrySize``        | 38              | 39              | 42                | 43               | 44               | 200                             |
```

Comment (JaredCorduan) [reply doc/explanations/min-utxo-alonzo.rst]: thank you for helping making this better! how did you verify this?

Comment (francolq) [reply doc/explanations/min-utxo-alonzo.rst]: Well, I did all the math, but you can verify that 200 multiplied by coinsPerUTxOWord (34,482) gives minUTxO (6,896,400).

Comment (JaredCorduan) [reply doc/explanations/min-utxo-alonzo.rst]: ah, of course, I forgot we had the 6,896,400 value, thanks!

**Review by JaredCorduan — APPROVED** [review]
thank you very much!

**Pull request merged**